tests/instruments/ptw/test_ptwUNIDOS.py

```
@@ -32,239 +32,328 @@
+            assert inst.measurement_result == {"status": "RES",
+                                               "charge": 2.6E-3,
+                                               "dose": 2.6E-3,
+                                               "current": 5.6E-9,
+                                               "doserate": 5.6E-9,
+                                               "timebase": "min",
+                                               "time": 3000,
+                                               "voltage": 300,
+                                               "error": "0x0"}
+
+    @pytest.mark.parametrize("range", RANGES)
+    def test_range(self, range):
```

Ruff (0.15.12)

[error] Function argument `range` is shadowing a Python builtin

(A002)